lib/taskana-core/src/main/java/pro/taskana/mappings/QueryMapper.java

```diff
@@ -274,48 +344,49 @@
         @Result(property = "custom16", column = "CUSTOM_16")})
     List<TaskSummaryImpl> queryTaskSummaries(TaskQueryImpl taskQuery);
```

Comment:
Would it be possible to split this into "queryTaskSummaries" and "queryTasksSummariesOptimizedForDb2"? This is getting too complex. And please add the sample query I've sent as a comment. This helps to understand the structure of the query.

Reply:
Done